torch/utils/checkpoint.py

```diff
@@ -477,7 +474,7 @@ def forward(input):
 # - If there are multiple .grad()/.backward() calls, we would perform backward
 #   on the recomputed graph even if early-stop is enabled (see the example below)
 #
-# [ Multiple backwards ]
+# [ retain_graph is False ]
```

Comment:
Use a simpler example to demonstrate the same issue since what really matters is that retain_graph=False, not that we have multiple backwards